easybuild/easyconfigs/a/ASTRID/ASTRID-2.2.1-gompi-2019a-Java-11.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'ASTRID'
+version = "2.2.1"
+versionsuffix = '-Java-%(javaver)s'
```

Comment:
@omula I'm sorry if I was not clear enough.
the versionsuffix is only used if the Java version differs from the default for a given toolchain generation. for `2019a` the default is Java 11, so you should not add the versionsuffix.

Reply:
@smoors changed!
Btw, how do I know which is the default Java version given a toolchain? Is there also a default Python version?
Thanks

Reply:
@omula thanks!

I don't know if there is an official list. I just grep the easyconfigs in the develop branch.
the first version of any software that is used as dependency is considered the default. other versions are not allowed, with exceptions for ['ASE', 'Boost', 'Java', 'Lua', 'PLUMED', 'R'].
so yes, there is also a default Python version :)